### PR TITLE
Human code refactor the sequel

### PR DESCRIPTION
**Pull request by RigglePrime**
Still April fools.

Oh god, what has Riggle done this time... It's the sequel. Same as before (#65839) but even worse. Somehow.

Warcrimes sponsored by Timberpoes and Domitius.

### CODE REVIEW

**Comment by RigglePrime** [comment]
Rock bottom.

**Comment by Rhials** [comment]
has april fools gone too far??

**Comment by interestingusernam3** [comment]
Ooh the misery

**Comment by itseasytosee** [comment]
A dark omen

File: code/modules/surgery/organs/tails.dm [hunk 1]
```diff
@@ -121,3 +121,57 @@
...
+		tail_owner.update_body()
+
+/obj/item/organ/tail/fox/alt
+	name = "fox tail"
+	desc = "A severed fox tail. Sad."
+	tail_type = "Fox 2"
+
+/obj/item/organ/tail/shark
+	name = "fox tail"
+	desc = "A severed fox tail. Sad."
```
Comment (diamondrel) [inline code/modules/surgery/organs/tails.dm:157]: Are these supposed to be fox tails?

Comment (RigglePrime) [reply code/modules/surgery/organs/tails.dm:157]: Damn it, forgot to change it. I'll update it

**Comment by Imaginos16** [comment]
Oh the misery, everybody wants to by my enemyyyyyy

**Comment by unit0016** [comment]
BASED.

**Comment by notfrying1pans** [comment]
unironically keep this

**Pull request closed without merging**